account_bank_statement_import_txt_xlsx/models/account_bank_statement_import_sheet_parser.py

```diff
@@ -51,9 +53,12 @@ def parse(self, data_file, mapping):
         if not lines:
             return currency_code, account_number, [{"transactions": []}]
 
-        lines = list(sorted(lines, key=lambda line: line["timestamp"]))
```

Comment:
Why did you remove sorting? Do you anticipate that data in CSV is always ordered by date?